testing/unittests/jlibtests.cpp

```diff
@@ -482,11 +483,33 @@ class JlibTraceTest : public CppUnit::TestFixture
                 CPPUNIT_ASSERT_MESSAGE("Mismatched localParentSpanID detected",
                  strsame(serverSpanID, retrievedSpanCtxAttributes->queryProp("localParentSpanID")));
 
-                CPPUNIT_ASSERT_EQUAL_MESSAGE("Unexpected missing remoteParentID detected", true,
-                 retrievedSpanCtxAttributes->hasProp("remoteParentID"));
+                CPPUNIT_ASSERT_EQUAL_MESSAGE("Unexpected GlobalID detected", false,
+                retrievedSpanCtxAttributes->hasProp(kGlobalIdHttpHeaderName));
+                CPPUNIT_ASSERT_EQUAL_MESSAGE("Unexpected CallerID detected", false,
+                retrievedSpanCtxAttributes->hasProp(kCallerIdHttpHeaderName));
 
-                CPPUNIT_ASSERT_MESSAGE("Unexpected CallerID detected",
-                 strsame(serverTraceID, retrievedSpanCtxAttributes->queryProp("remoteParentID")));
+                CPPUNIT_ASSERT_EQUAL_MESSAGE("Unexpected Declared Parent SpanID detected", false,
+                retrievedSpanCtxAttributes->hasProp("remoteParentSpanID"));
+
+                CPPUNIT_ASSERT_EQUAL_MESSAGE("Unexpected empty TraceID detected", false, isEmptyString(retrievedSpanCtxAttributes->queryProp("traceID")));
+                CPPUNIT_ASSERT_EQUAL_MESSAGE("Unexpected empty SpanID detected", false, isEmptyString(retrievedSpanCtxAttributes->queryProp("spanID")));
+            }
+        }
+
+        {
+            unsigned startMs = msTick() - 1234;
+            SpanTimeStamp SpanTS;
+            SpanTS.setMSTickTime(startMs);
+            OwnedSpanScope clientSpan = serverSpan->createClientSpan("clientSpan", &SpanTS);
```

Comment:
L:503 exercises the new constructor signature, but the bulk of this test seems unrelated to the functionality we want to test which is the span's manually declared start time. I think manualTestsDeclaredSpanStartTime could be used as a template to test this new functionality